patches/roslyn-analyzers/0001-WIP-for-source-build.patch

```diff
@@ -0,0 +1,1087 @@
+-		HideSolutionNode = FALSE
+-	EndGlobalSection
+-	GlobalSection(NestedProjects) = preSolution
+-		{FF60CC25-31F6-4FE8-8C62-256F0B117353} = {4889919F-291D-4909-911C-21B9005AE794}
+-		{EC946164-1E17-410B-B7D9-7DE7E6268D63} = {1F4F7A9B-FD3B-495F-86D0-89A7DEA2128C}
+-		{F89AD21E-42A4-4460-9B85-B9CFF295BD49} = {4ABFCD94-20BB-48A4-B9CD-EE304812F03B}
+-		{CB7A5E66-902D-49C2-A952-8989EBFABD8D} = {4ABFCD94-20BB-48A4-B9CD-EE304812F03B}
+-		{99F594B1-3916-471D-A761-A6731FC50E9A} = {1F4F7A9B-FD3B-495F-86D0-89A7DEA2128C}
+-		{FCB56CBA-FA35-46A8-86B7-BAE5433197D9} = {1F4F7A9B-FD3B-495F-86D0-89A7DEA2128C}
+-		{DB2A3703-0149-46D4-B575-018346B2F406} = {02441226-0D01-4ED2-A1F6-46BB956307FF}
+-		{FEE46B63-96AC-476E-80C3-4CA964E748E9} = {9201CFE0-167E-447E-B7EF-933341A3797D}
+-		{E038E362-EFBE-4A81-A93F-CFA5D779FD2E} = {31B73922-AADE-4B0C-88F5-92AA5883A0E6}
+-		{BCB20922-349F-45A6-A8CA-0BC9C29C878C} = {02441226-0D01-4ED2-A1F6-46BB956307FF}
+-		{47A90FE9-1942-44AD-ABBC-AF4AD45C04B2} = {9201CFE0-167E-447E-B7EF-933341A3797D}
+-		{DB4AAE94-5DEC-4CCB-91C4-1244D72E7966} = {31B73922-AADE-4B0C-88F5-92AA5883A0E6}
+-		{748416D8-6D6C-4F85-B29A-E145DE589715} = {02441226-0D01-4ED2-A1F6-46BB956307FF}
+-		{EC0535DF-6A80-428A-B13A-C0FD22AE4592} = {9201CFE0-167E-447E-B7EF-933341A3797D}
+-		{A410FBFA-994E-450B-A492-177D20A7D9FE} = {31B73922-AADE-4B0C-88F5-92AA5883A0E6}
+-		{E75CA893-D29C-43F3-A262-E0DF90A8FE00} = {02441226-0D01-4ED2-A1F6-46BB956307FF}
+-		{4367C446-0EB2-4EFC-B9C3-8A0C3CA34E02} = {9201CFE0-167E-447E-B7EF-933341A3797D}
+-		{3AA1978C-31A1-4002-8A21-5A2560DD95B8} = {31B73922-AADE-4B0C-88F5-92AA5883A0E6}
+-		{89B4A206-72E8-4DF6-8C59-1AB4E00DCDC5} = {4889919F-291D-4909-911C-21B9005AE794}
+-		{EFF69ED7-A8AB-4394-BDAE-9C4A1A19D150} = {02441226-0D01-4ED2-A1F6-46BB956307FF}
+-		{95519BE2-5ED6-466A-9EB4-A1152950A567} = {9201CFE0-167E-447E-B7EF-933341A3797D}
+-		{16C5B8A9-E410-45E9-8F6A-583BF459F09C} = {31B73922-AADE-4B0C-88F5-92AA5883A0E6}
+-	EndGlobalSection
+-	GlobalSection(ExtensibilityGlobals) = postSolution
+-		SolutionGuid = {FC44ACA9-AEA3-4EE6-881C-2E08ED281B5F}
+-	EndGlobalSection
+-EndGlobal
+diff --git a/nuget.config b/NuGet.config
+similarity index 100%
+rename from nuget.config
+rename to NuGet.config
+diff --git a/RoslynAnalyzers.sln b/RoslynAnalyzers.sln
+index 82e87f32e..48de016c5 100644
+--- a/RoslynAnalyzers.sln
++++ b/RoslynAnalyzers.sln
+@@ -10,8 +10,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Roslyn.Diagnostics.CSharp.A
+ EndProject
+ Project("{778DAE3C-4631-46EA-AA77-85C1314464D9}") = "Roslyn.Diagnostics.VisualBasic.Analyzers", "src\Roslyn.Diagnostics.Analyzers\VisualBasic\Roslyn.Diagnostics.VisualBasic.Analyzers.vbproj", "{70BBA457-2CC1-4929-8FEE-359EBB7C398A}"
+ EndProject
+-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Roslyn.Diagnostics.Analyzers.UnitTests", "src\Roslyn.Diagnostics.Analyzers\UnitTests\Roslyn.Diagnostics.Analyzers.UnitTests.csproj", "{D90E7402-70E7-4A95-A292-47EA163D6309}"
```

Comment:
I have removed all Unit Test projects. Not sure if we should remove fewer or more projects.